modules/machine-edge-pool-review-nodes.adoc

```diff
@@ -6,7 +6,8 @@
 [id="machine-edge-pool-review-nodes_{context}"]
 = Verifying nodes that were created with edge compute pool
 
-After you install a cluster that uses AWS Local Zones, check the status of the machine that was created by the machine set manifests created at install time.
+After you install a cluster that uses {aws-service-name-full}, check the status of the
```

Comment:
Uses {aws-service-name-full} infrastructure would be clearer?

Reply:
yes, it would be better